packages/governance/src/contractGovernance/governApi.js

```diff
@@ -28,17 +28,13 @@ const makeApiInvocationPositions = (apiMethodName, methodArgs) => {
 /**
  * manage contracts that allow governance to invoke functions.
  *
- * @param {ERef<ZoeService>} zoe
- * @param {Instance} governedInstance
  * @param {ERef<{ [methodName: string]: (...args: any) => unknown }>} governedApis
  * @param {Array<string | symbol>} governedNames names of the governed API methods
  * @param {ERef<import('@agoric/time/src/types').TimerService>} timer
  * @param {() => Promise<PoserFacet>} getUpdatedPoserFacet
- * @returns {Promise<ApiGovernor>}
+ * @returns {ApiGovernor}
  */
-const setupApiGovernance = async (
-  zoe,
-  governedInstance,
```

Comment:
these weren't used? how did that pass lint?

Reply:
because the subsequent params were used, they had to stay there for backwards compatibility